assignments/005-slurm-and-bwa-mem2/index2.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#SBATCH --cpus-per-task=1
+#SBATCH --time=0:15:00
+#SBATCH --mem=3G
+#SBATCH --job-name=fastp_trim_all
+#SBATCH --output=fastp_trim_all-%j.out
+#SBATCH --error=fastp_trim_all-%j.err
+
+####  Mike Antolin
+####  005-005-slurm-and-bwa-mem2/index2.sh
+
+####  Get ALPINE ready to SLURM
+module load slurm/alpine
+
+#### get a node
+acompile -h
+
+####  Make sure I'm in the right level
+cd /projects/antolin@colostate.edu/con-gen-csu
```

Comment:
You can do this in the script, but you can also just run sbatch in the correct directory.  The current working directory gets inherited by sbatch on SLURM on Alpine.